### DISCUSSION TITLE
UI ERROR

### DISCUSSION THREAD

**Discussion by redolf250** [rabbitmq/rabbitmq-server#10714] [Other] [answered]
### Describe the bug

![rabbit](https://github.com/rabbitmq/rabbitmq-server/assets/55606159/8efc07a5-1958-42ae-89cf-8da955692657)

### Reproduction steps

Install the version as seen from the 3.12.3, I also tried 3.13.0
It showed the same error

### Expected behavior

The UI show work fine

**Comment by michaelklishin** [comment, marked as answer]
This question has been beaten to death. You need to [clear your browser's cache](https://www.rabbitmq.com/docs/upgrade#management-plugin-upgrades) after upgrading. Management UI is an old app that does not have sophisticated Web asset packaging.

**Comment by redolf250** [comment]
Thanks for the response. I’ve switched to a different browser, it works
fine now.
